package.json

```diff
@@ -10,7 +10,17 @@
     "add:mutation": "node ./src/scripts/addMutation.js",
     "build": "rm -rf dist && tsc",
     "generate-docs": "./deploy-doc.sh",
-    "prepublish": "rm -rf dist && tsc"
+    "prepublish": "npm run transpile && npm run build",
+    "commen1": "############################## Cheapreats Package Install ##############################",
```

Comment:
What.

Reply:
oh Comment lol

Reply:
Can't you use
//
or 
# 
or /* */ to comment?

Reply:
No, it's a JSON file and that's it, so if you want comments this is the only way unfortunately.

Reply:
Then use
*_comment*